hathor/prometheus.py

```diff
@@ -212,6 +213,8 @@ def _set_garbage_collection_metrics(self) -> None:
         threshold = gc.get_threshold()
 
         for i in range(3):
+            self.gc_metrics['tracked_objects'].labels(generation=i).set(len(gc.get_objects(i)))
+            # This metric is called `objects` for historical reasons, to keep compatibility with existing graphs.
```

Comment:
I think we can change it and then update the charts. The updated there is trivial.